content/Developers_Handbook/Kafka/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Kafka
+weight: 11
+---
+
+## Read messages from Kafka
+
+### Number of partitions and their message count 
+
+Substitute `dataset-events` with the desired topic.
+
+```bash
+kubectl exec -it kafka-1-0 -- /bin/kafka-run-class kafka.tools.GetOffsetShell --broker-list localhost:9092 --topic dataset-events
+```
+
+#### Example output
+
+```txt
+dataset-events:0:47472
+dataset-events:1:49949
+dataset-events:2:49361
+dataset-events:3:50122
+```
+
+### Read last n messages from partition in topic
+
+Substitute _topic_, _partition_ and _offset_ parameters based on output from command above.
+Remember that all messages with the same key (often some sort of ID) is put on the same partition.
```

Comment:
Hva er/hva gjør _topic_-, _partition_- og _offset_-parametrene?

Reply:
Kanskje vi skal lenke til en kafka 101, så man kan lese den først?
https://medium.com/@c.ramprasad273/kafka-101-introduction-to-kafka-770887a46d32

Reply:
![giphy](https://user-images.githubusercontent.com/11135068/206122448-10a924dd-346e-427b-b11d-8699ff9ff62e.gif)